docs/team/jehousoh.md

```diff
@@ -5,42 +5,68 @@ title: Soh Je Hou's Project Portfolio Page
 
 ### Project: GuestBook
 
-GuestBook is a one stop access to get all the necessary guest data.
+GuestBook is a **desktop app for managing guests in a hotel,
+optimized for use via a Command Line Interface** (CLI)
+while still having the benefits of a Graphical User Interface (GUI).
+If you are a hotel manager who can type fast, GuestBook can get your
+guest management tasks done faster than traditional GUI apps.
 The user interacts with it using a CLI, and it has a GUI created with JavaFX.
-It is written in Java, and has about 10 kLoC.
+It is written in Java, and has about 13 kLoC.
 This project is based off AddressBook - Level 3.
 
 Given below are my contributions to the project.
 
-* **New Feature**:
-  * to be added soon
+* **New Feature**: Added `NumberOfGuests` field.
 
 * **Code contributed**: [[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=jehousoh&tabRepo=AY2223S1-CS2103T-W16-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)]
 
-* **Project management**:
-  * to be added soon
+* **Enhancements to existing features**: Enhanced `find` command to allow for the searching all fields.
+  * This feature increases the convenience for hotel managers to search for their guests by allow the search of other
+    fields (eg. Room, date of stay).
 
-* **Enhancements to existing features**:
-  * to be added soon
+* **Enhancements to existing features**: Improved the user interface (UI).
+  * Added icons to each field to be more concise yet remain clear.
+  * Categorised fields by columns based on type of information.
+  * Updated application icon.
+  * Updated isRoomClean to a green/red icon signifying yes/no respectively.
+  * Enable text wrapping.
+  * Allow Ui to be resizable.
 
 * **Documentation**:
   * User Guide:
-    * to be added soon
+    * Added "Components of the UI" section.
+    * Added UiComponentsLabels.png.
+    * Added acknowledgements for icons.
```

Comment:
Should the acknowledgement be under DG instead of UG?

Reply:
Good catch! Changed.